docs/events.yaml

```diff
@@ -410,3 +410,231 @@ x-events:
   severity: "info"
   properties:
     infra_env_id: UUID
+
+- name: discovery_ignition_config_applied
+  message: "Custom discovery ignition config was applied to the cluster"
+  event_type: cluster
+  severity: "info"
+  properties:
+    cluster_id: UUID
+
+- name: image_deleted_after_ignition_update
+  message: "Deleted image from backend because its ignition was updated. The image may be regenerated at any time"
+  event_type: cluster
+  severity: "info"
+  properties:
+    cluster_id: UUID
+
+- name: download_image_failed_not_found
+  message: "Failed to download image: the image was not found (perhaps it expired) - please generate the image and try again"
+  event_type: infra_env
+  severity: "error"
+  properties:
+    infra_env_id: UUID
+
+- name: download_image_started
+  message: "Started image download (image type is '{image_type}')"
+  event_type: infra_env
+  severity: "info"
+  properties:
+    infra_env_id: UUID
+    image_type: string
+
+- name: generate_image_failed_start
+  message: "Failed to generate image: error starting DB transaction"
+  event_type: cluster
+  severity: "error"
+  properties:
+    cluster_id: UUID
+
+- name: generate_image_failed_contact_storage_backend
+  message: "Failed to generate image: error contacting storage backend"
+  event_type: cluster
+  severity: "error"
+  properties:
+    cluster_id: UUID
+
+- name: generate_image_failed_update_metadata
+  message: "Failed to generate image: error updating metadata"
+  event_type: cluster
+  severity: "error"
+  properties:
+    cluster_id: UUID
+
+- name: generate_image_failed_commit_transaction
```

Comment:
This event gets used in GenerateClusterISOInternal
() , therefore it should also include the infra_env_id field.

GenerateClusterISOInternal() runs only if it can find a valid infra_env_id:
https://github.com/openshift/assisted-service/blob/8083e57a0642fe0aec81342547db9a30114a2583/internal/bminventory/inventory.go#L1184-L1188